organizers/2018-12-17.html

```diff
@@ -0,0 +1,219 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>December 17, 2018</h2>
+    <img data-src="/img/django-logo-white-transparent.svg" style="width:40%; background: none" alt="Django logo" class="plain">
+</section>
+
+<section>
+    <h1>Welcome All!</h1>
+    <h2>🎄 Happy Holidays! 🍾</h2>
+    <h2>👋😄</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Time Out</strong></li>
+        <li><strong>Verve</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain">
+    <p>
+        <em>Europe's largest music festival platform, and UK's 6th fastest growing private tech start up.<br>Transforming the way music fans discover, book tickets and travel packages for festivals around the world.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain">
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/verve.svg" style="width:40%" alt="Verve" class="plain">
+    <p>
+        <em>We turn word of mouth into your best sales channel.</em>
+    </p>
+</section>
+
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>📅 2018 Year Review</h2>
+    <h2>💰 Sponsors</h2>
+    <h2>🎤 Talk</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>❓ Quiz</h2>
+    <h2>🍕 Food &amp; Drink</h2>
+</section>
+
+
+<section>
+    <h1>📅  2018 Year Review</h1>
+    <h2>Our amazing speakers!</h2>
+    <p>Aaron Kirkbride, Adam Charnock, Adam Malinowski, Antonio Melé, Charalampos Papaloizou, Chris Wedgwood, Christopher McDonald, Craig Anderson, Dan Palmer, David Heath, David Seddon, Ibukun Oluwayomi, Lili Kastilio, Luna Chen, Mat Clayton, Matthew Hall, Nicholas Herriot, Patrick Fitzgerald, Rob Kirberich, Rolo Mawlabaux, Silvian Dragan.</p>
+</section>
+
+
+<section>
+    <h1>Speakers diversity  </h1>
+    <h2>We want more speaker diversity</h2>
+    <h2>👩🏻‍💻 👨🏽‍💻 👩🏿‍💻 👨🏻‍💻 👩🏽‍💻 👨🏿‍💻 </h2>
+    <p>In 2018 only 2 female speakers: Lili Kastilio and Luna Chen</p>
+    <p>Long term goal is to have 50/50</p>
+</section>
+
+
+<section>
+    <h1>📅  2018 Year Review</h1>
+    <h2>New organisers</h2>
```

Comment:
James, David, and myself weren't new this year, so perhaps just "Organisers" as the title?